app/api/user.py

```diff
@@ -56,10 +78,13 @@ def register():
 def login():
```

Comment:
why we are taking three fields in login there should be only email and password....